player/player.gd

```diff
@@ -19,7 +19,7 @@ var _ammo: int
 var _max_ammo: int
 
 var _is_reloading: bool = false
-var _reload_time: float = 3.0
+var _reload_time: float = 2.5
```

Comment:
might be better to just use reload time as a time to reload one bullet instead the entire clip.

with the way things are implemented now, increasing the clip size will increase reload speed since no matter how many bullets need to be reloaded, it will only take 2.5 seconds.